translations/README_es.md

```diff
@@ -0,0 +1,1916 @@
+    - [Aprendizaje automático](#aprendizaje-automático)
+- [Detalles adicionales de ciertos temas](#detalles-adicionales-de–ciertos-temas)
+- [Series de videos](#series-de-videos)
+- [Cursos de Informática](#cursos-de-informática)
+
+---
+
+## ¿Por qué usarlo?
+
+Cuando comencé este proyecto, no sabía la diferencia entre un stack y un heap, no conocía la notación Big-O, nada acerca de árboles, ni cómo sacar la transversal de una gráfica. Si tenía que programar un algoritmo de clasificación, puedo decir que no hubiera sido muy bueno. Cada estructura de datos que había utilizado estaba incorporada al lenguaje, y yo no sabía cómo funcionaban realmente. Yo nunca tuve que manejar la memoria a menos que un proceso que yo estaba corriendo diera un error de “out of memory”, y tenía que encontrar una alternativa. He usado pocos arreglos de varias dimensiones en mi vida y miles de arreglos asociativos, pero nunca he creado estructuras de datos desde cero.
+
+Pero después de pasar por todo este plan de estudios tengo mucha confianza de que seré contratado. Me toará meses. Si mucho de esto te resulta familiar entonces te tomará mucho menos tiempo.
+
+Es un plan largo, puede tome unos meses; pero si le resulta familiar mucho de esto le tomara mucho menos tiempo.
+
+## Como usarlo
+
+Todo lo que aparece abajo es un plan, y debería abordar los elementos en orden de arriba a abajo.
+
+Estoy usando el tema especial de markdown de Github, incluyendo listas de tareas para comprobar el progreso.
+
+[] **Cree una nueva rama para que pueda validar elementos como este, solo ponga una x en los corchetes: [x]**
+
+
+    Haga fork a una rama y siga los comandos siguientes
+
+`git checkout -b progress`
+
+`git remote add jwasham https://github.com/jwasham/coding-interview-university`
+
+`git fetch --all`
+
+    Marque todas las cajas con una x después de que completa los cambios
+
+`git add . `
+
+`git commit -m "Marked x" `
+
+`git rebase jwasham/master `
+
+`git push --force `
+
+[Más acerca de los temas de Github para markdown](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown)
+
+## No sienta no es lo suficientemente listo
+- Los ingenieros de software exitosos son inteligentes pero muchos tienen la inseguridad de no ser lo suficientemente listos.
+- [The myth of the Genius Programmer](https://www.youtube.com/watch?v=0SARbwvhupQ)
+- [It's Dangerous to Go Alone: Battling the Invisible Monsters in Tech](https://www.youtube.com/watch?v=1i8ylq4j_EY)
+
+## Acerca de los recursos de vídeo
+Algunos videos solo están disponibles inscribiéndose a una clase de Coursera, EdX, o Lynda.com. Estos son llamados MOOCs.
+En ocasiones las clases no están en sesión por lo que tendrá que esperar un par de meses, por lo que no tiene acceso. Los cursos de Lynda.com no son gratuitos.
+
+Apreciaría su ayuda añadiendo fuentes siempre disponibles, publicas y gratuitas como videos de Youtube para acompañar a los videos del curso en línea.

```

Comment:
This paragraph needs to match the indentation of the next one to be in the same grey box